src/app/search/search-results/search-results.component.ts

```diff
@@ -20,28 +20,31 @@ import { ELASTIC_SEARCH_CLIENT } from '../elastic-search-client';
 import { QueryBuilderService } from '../query-builder.service';
 import { SearchQuery } from '../state/search.query';
 import { SearchService } from '../state/search.service';
+import { Base } from '../../shared/base';
+import { map } from 'rxjs/operators';
 
 @Component({
   selector: 'app-search-results',
   templateUrl: './search-results.component.html',
   styleUrls: ['./search-results.component.scss']
 })
-export class SearchResultsComponent implements OnInit {
-  public browseToolsTab = 'browseToolsTab';
-  public browseWorkflowsTab = 'browseWorkflowsTab';
-  public activeToolTab$: Observable<boolean>;
+export class SearchResultsComponent extends Base implements OnInit {
+  public activeToolTab$: Observable<number>;
   public noToolHits$: Observable<boolean>;
   public noWorkflowHits$: Observable<boolean>;
   public showWorkflowTagCloud$: Observable<boolean>;
   public showToolTagCloud$: Observable<boolean>;
+  public selectedIndex$: Observable<any>;
```

Comment:
This would ideally not be `any`. Or were you forced to do it due to some compiler issues?

Reply:
IntelliJ would complain if I got rid of any as selectedIndex$ is type Observable<{active: number;}>. I believe that it might be best to leave it as type any for now.

Reply:
Hmm, that works for me. Where did it complain?

Reply:
Line 58. Once you change the type from any to number or anything else, it will complain.

Reply:
I tried it and did a build, and it worked. Might be an IDE configuration thing.